IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -60,7 +62,7 @@
                         <td>@brr.Longitude</td>
                         <td>@brr.Latitude</td>
                         <td>@brr.Phone</td>
-                        <td>@brr.WebsiteUrl</td>
+                        <td><a href="@brr.WebsiteUrl" target="_blank">@brr.WebsiteUrl</a></td>

```

Comment:
In the existing website there are no hyperlinks for wherever link for website is mentioned. So, I changed all the available links to hyperlink.
https://www.w3schools.com/html/html_links.asp